docs/usage/examples/self-hosting.md

```diff
@@ -166,7 +166,7 @@ The following example runs Renovate hourly, and looks for the self-hosted config
 ```yml
 version: '2.1'
 orbs:
-  renovate: daniel-shuy/renovate@2.1.1
+  renovate: daniel-shuy/renovate@2.2.0
```

Comment:
Note: there's an `3.x` release line of this Orb now, but that comes with breaking changes.

Do we want an issue to track the rework of the documentation to account for the breaking changes?

Reply:
Yes, sounds good